python/llm/example/CPU/HF-Transformers-AutoModels/Model/qwen1.5/README.md

```diff
@@ -0,0 +1,87 @@
+# Qwen1.5
+
+In this directory, you will find examples on how you could apply BigDL-LLM INT4 optimizations on Qwen1.5 models. For illustration purposes, we utilize the [Qwen/Qwen1.5-7B-Chat](https://huggingface.co/Qwen/Qwen1.5-7B-Chat) as a reference Qwen1.5 model.
+
+## 0. Requirements
+To run these examples with BigDL-LLM, we have some recommended requirements for your machine, please refer to [here](../README.md#recommended-requirements) for more information.
+
+## Example: Predict Tokens using `generate()` API
+In the example [generate.py](./generate.py), we show a basic use case for a Qwen model to predict the next N tokens using `generate()` API, with BigDL-LLM INT4 optimizations.
+### 1. Install
+We suggest using conda to manage environment:
+```bash
+conda create -n llm python=3.9
+conda activate llm
+
+pip install --pre --upgrade bigdl-llm[all] # install bigdl-llm with 'all' option
+pip install transformers==4.37.2 # install the transformers which support Qwen2
```

Comment:
Let's user transformers 4.37.0? qwen2_modelling.py did not changed from 4.37.0

Reply:
Sure, I will test the qwen1.5 on transformers 4.37.0